cmd/build.rs

```diff
@@ -32,7 +32,7 @@ fn link_sys_lib(lib: &str, tool: &cc::Tool) {
     }
     // remove lib prefix and .a postfix.
     let libname = &lib[3..lib.len() - 2];
-    println!("cargo:rustc-link-lib=static={}", &libname);
+    println!("cargo:rustc-link-lib=static:+whole-archive={}", &libname);
```

Comment:
what's this

Reply:
From what I can read, whole-archive flag will force using static symbols when there're both a static and a dynamic version available. It is enabled by default before. The fact that we must enable it to pass the check means we somehow linked multiple versions of libstdc++.

Reply:
How can that happen? I think it should report duplicated symbol error.

Reply:
I meant to say that multiple versions are available for the linker. So it's up to the implementation to choose one of them. And FYI during the debugging I have ruled out rust-rocksdb by setting the `static_libcpp` feature.